src/cm.css

```diff
@@ -68,6 +68,25 @@
   font-weight: 400;
 }
 
+
+/***********************************
+ * Accessibility - non-visual label
+ ***********************************/
+
+ /** Screen reader styles */
+ .screen-reader {
```

Comment:
Am I missing something? Nothing really changed except for the location of this block?